cdap-app-fabric/src/test/java/co/cask/cdap/internal/AppFabricClient.java

```diff
@@ -239,6 +248,51 @@ public Instances getFlowletInstances(String namespaceId, String applicationId, S
     return schedules;
   }
 
+  public WorkflowTokenDetail getWorkflowToken(String namespaceId, String appId, String wflowId, String runId,
```

Comment:
most stuff other than the code that sets defaults for scope and key is different, no? Necessary to refactor?